scripts/Linux-Pkg/kathara-aur/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+pkgname=kathara
+pkgver=__VERSION__
+pkgrel=__PACKAGE_VERSION__
+pkgdesc="Lightweight network emulation system based on Docker containers."
+arch=('x86_64')
+url="https://www.kathara.org/"
+license=('GPL3')
+install="kathara.install"
+makedepends=(
+  'python>=3.5.0' 
+  'python-pip' 
+  'nuitka'
+  'ruby-ronn-ng'
+)
+optdepends=(
+  'docker: for running network scenarios in a local environment'
+  'xterm: for opening devices terminals'
+)
+source=("${pkgname}-${pkgver}::https://github.com/KatharaFramework/Kathara/archive/${pkgver}.tar.gz")
```

Comment:
We can try to partially compile the Nuitka stuff locally and then push only the compiled code to AUR

Reply:
The reason why I was compiling it from source is to minimize compatibility issues. You can still do that, but at that point Kathara will be only amd64 compatible, at least initially. If it is ok with you I can work with it.